bihire/team_work_server
# feat(server): version my api

### What does this PR do?
- Version your API using URL versioning starting with the letter “v”
### Description of task to be completed?
- dynamically redirect the user to version
- default version message if version is not specified
### How should this be manually tested ?
after cloning this repository
- open a terminal in the directory
- run the server by "npm start"
- open postman and hit this url "localhost:8080/api/v1" as POST
- should receive a message
### What are the relevant PivotTracker stories?
#168387137
